src/Raven.Server/Documents/Sharding/Operations/ShardedQueryOperation.cs

```diff
@@ -197,6 +185,16 @@ static IEnumerator<BlittableJsonReaderObject> GetEnumerator(BlittableJsonReaderA
         return result;
     }
 
+    private class RoundRobinComparer : IComparer<BlittableJsonReaderObject>
```

Comment:
What is the primary purpose of having such comparer?

Reply:
I need some default comparer to unify the code.
And I thought when we will have this impl. with streaming rather than materialize the entire results, it might be better to read each time from a different stream.